lib/embed/media_tag.rb

```diff
@@ -94,7 +94,7 @@ def enabled_streaming_sources(file)
     end
 
     def transcript(file)
-      return unless file.vtt
+      return if !file.vtt || !Settings&.enabled_features&.transcripts
```

Comment:
Why do you have the safe navigation operator here? There's no way for these to be nil, is there?

What's more, is that I'd prefer this value to be injected into the initializer of MediaTag.  Thus ```initialize(viewer, display_transcript: false)```

This allows you to remove `set_transcripts_flag_per_param` from the controller and pass `MediaTag.new(viewer, params[:transcript] == 'true')` in https://github.com/sul-dlss/sul-embed/blob/e1194564101d7e6ec209d920bd1c457d9e6a8fdb/app/views/embed/body/_media.html.erb#L4

Reply:
I moved it to the constructor, with a named parameter. Let me know if that doesn't make sense, and thanks for this suggestion.